login-screen.html

```diff
@@ -0,0 +1,11 @@
+<html>
+
+<head>
+  <title>Login screen</title>
+</head>
+
+<body>
+  Login screen. With more functionality.
+</body>
```

Comment:
Shouldn't this be </buddy> ?

Reply:
I think you mean </booty>

Reply:
Great point, I thought about that earlier, but I think `</body>` is actually what want here for reasons...

Reply:
I disagree.  ```
<buddy />``` is cooler